### ISSUE TITLE
Do not open already minimized review comments

### ISSUE THREAD

**Issue by Thomas1664** [microsoft/vscode-pull-request-github#3752]
I'm not sure weither this is a bug or a feature request.

Every time I switch to an editor all review comments that I previously minimized are poppig up again.
I would like to see that my choice to minimize all comments in a file would be remembered.

The current experience for me is very annoying because I can't read the code fluently if there are many comments and I always have to click the minimize button when I change to a file that contains comments.

The workaround for me was to set the setting `githubPullRequests.commentExpandState` to `collapseAll` which is not what I want.

**Comment by alexr00** [comment]
@Thomas1664 Is this in a PR that you have checked out or a PR that you're reviewing from the "Pull Requests" view?

**Comment by Thomas1664** [comment]
In a PR that I have checked out.

**Comment by alexr00** [comment]
Thank you for the additional info. We should be persisting your comment expanded state when you switch between editors.

**Comment by alexr00** [comment]
@Thomas1664 I'm not able to reproduce this issue. Does this occur when you just switch back and forth between two files?

**Comment by Thomas1664** [comment]
> @Thomas1664 I'm not able to reproduce this issue. Does this occur when you just switch back and forth between two files?

Yes.
I noticed the issue specifically in this PR: https://github.com/microsoft/vcpkg-tool/pull/636 (although only one comment is left).
I removed my workaround setting and tried to reproduce it as well but I couldn't. Possible reasons are that I used a codespace this time, a VS Code update silently fixed the issue, or that there is something that both of us were missing while trying to reproduce it.
I suggest that I remove the workaround permanently and reach back out to you if I'm experiencing the issue again.
